ui/src/components/filter/permission.js

```diff
@@ -1,6 +1,10 @@
-export default (accessType, action, permissions) => {
-  if (permissions[accessType].includes(action)) {
-    return true;
-  }
-  return false;
-};
+// import { authorizer } from '../../authorizer';
+/**
+ *
+ * @param accessType
+ * @param action
+ * @param permissions
+ * @returns {boolean}
+ */
+// eslint-disable-next-line max-len
```

Comment:
Could it to be avoided?

Reply:
Yes, it does, however…

The ESLInt rule, arrow-body-style, request to the function, when as simple as that, to be in a line, but, when put in line, it breaks another ESLint rule, max-line.

Aiming for the simplicity, I have chosen to disable the max-line instead doing a return true and false block.